client/src/app/pages/dashboard/maps/map-info/map-leaderboard/map-leaderboard.component.html

```diff
@@ -19,13 +19,18 @@ <h2 class="col">LEADERBOARD</h2>
             <input nbInput fullWidth type="text" class="form-control" placeholder="Name">
           </div>
           <div class="col-md">
-            <nb-checkbox class="mr-1">W-Only</nb-checkbox>
-            <nb-checkbox class="mr-1">Backwards</nb-checkbox>
+            <!-- <nb-checkbox class="mr-1">W-Only</nb-checkbox>
+            <nb-checkbox class="mr-1">Backwards</nb-checkbox> -->
```

Comment:
You can remove things instead of commenting them out, we have version control and can always go back to get the old code.